db/migrate/20170327131238_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+class DeviseCreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+      # created extra boolean whether user is admin or not
+      t.boolean :admin,             default: false
```

Comment:
Please create test to expose/ensure that a user won't be an admin by default (as it is important business logic). Also, add a test to show your colleagues how to add an admin user to the system.